### PR TITLE
#SB-30225 feat: Enable User selection functionality

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

File: src/app/services/data/data.service.ts [hunk 1]
```diff
@@ -150,4 +155,19 @@ export class DataService {
         return observableOf(data);
       }));
   }
+
+  private getHeader(headers?) {
+    const default_headers = {
```
Comment (vaibhavbhuva) [inline src/app/services/data/data.service.ts:160]: @santoshilimi Don't pass it headers in the `data.services.ts` file.  Please check for another solution provide by angular.

Comment (santoshilimi) [reply src/app/services/data/data.service.ts]: Updated with changes

**Review by vaibhavbhuva — APPROVED** [review]
@vinukumar-vs I have reviewed it and looks fine for me.
Please review it once at your end and merge the PR.

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class HomeComponent implements OnInit {
   }
 
   navigateToEditors(){
-    this.router.navigate(['/editors']);
+    this.router.navigate(['/users']);
```
[the remark this answers is not in the document]
Comment (vaibhavbhuva) [reply src/app/home/home.component.ts:18]: @santoshilimi Please make sure function and variable names should be proper as per changes whenever you are modifying the existing code.

**Review by vaibhavbhuva — APPROVED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**